kernel/src/actions/mod.rs

```diff
@@ -24,7 +24,11 @@ use serde::{Deserialize, Serialize};
 pub mod deletion_vector;
 pub mod set_transaction;
 
+#[cfg(feature = "developer-visibility")]
+pub mod schemas;
```

Comment:
I suppose we need to make everything `pub` for the doctest stuff to work? I understand the need but want to flag as a downside..

Reply:
We _could_ check for `doctest` instead of `developer-features` -- but as @r3stl355 noted already, we use the latter already in so many places that this is the less invasive approach?
(alternatively, we could pursue the `doctest` feature gating, and use cfg any for the cases that overlap with `developer-features`)?